From 7afb0b3f496207a4894cdc81671c822c7a1ad4a6 Mon Sep 17 00:00:00 2001
From: Arthur Liberman <arthur_liberman@hotmail.com>
Date: Thu, 9 Jan 2025 00:06:41 +0200
Subject: [PATCH 05/17] Revert "AMLUtils: force mode switch by drm property
 'color_force'"

This reverts commit 7b09672537df9204d6ab80bdb8d78e6abbe1e7de.
---
 xbmc/utils/AMLUtils.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/xbmc/utils/AMLUtils.cpp b/xbmc/utils/AMLUtils.cpp
index 12052c7273..65e110b939 100644
--- a/xbmc/utils/AMLUtils.cpp
+++ b/xbmc/utils/AMLUtils.cpp
@@ -1335,7 +1335,7 @@ bool aml_set_drmDevice_mode(unsigned int width, unsigned int height, std::string
       drmModeFBPtr drm_fb = drmModeGetFB(fd, crtc->buffer_id);
 
       if (force_mode_switch)
-        set_drmProp(fd, connector->connector_id, "color_force", DRM_MODE_OBJECT_CONNECTOR, 1, NULL);
+        drmModeSetCrtc(fd, crtc->crtc_id, 0, 0, 0, NULL, 0, NULL);
 
       ret = drmModeSetCrtc(fd, crtc->crtc_id, drm_fb->fb_id, 0, 0,
         resources->connectors, 1, &connector->modes[i]);
-- 
2.48.1

